posthog/clickhouse/migrations/0139_drop_session_replay_events_v2_test_table.py

```diff
@@ -0,0 +1,10 @@
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+
+# NB the order of operations are important here
+operations = [
+    run_sql_with_exceptions("DROP TABLE IF EXISTS session_replay_events_v2_test_mv"),
+    run_sql_with_exceptions("DROP TABLE IF EXISTS kafka_session_replay_events_v2_test"),
+    run_sql_with_exceptions("DROP TABLE IF EXISTS sharded_session_replay_events_v2_test"),
+    run_sql_with_exceptions("DROP TABLE IF EXISTS writable_session_replay_events_v2_test"),
+    run_sql_with_exceptions("DROP TABLE IF EXISTS session_replay_events_v2_test"),
+]
```

Comment:
Small change to drop the distributed table from the coordinators as well 🙂 

```suggestion
operations = [
    run_sql_with_exceptions("DROP TABLE IF EXISTS session_replay_events_v2_test_mv"),
    run_sql_with_exceptions("DROP TABLE IF EXISTS kafka_session_replay_events_v2_test"),
    run_sql_with_exceptions("DROP TABLE IF EXISTS sharded_session_replay_events_v2_test"),
    run_sql_with_exceptions("DROP TABLE IF EXISTS writable_session_replay_events_v2_test"),
    run_sql_with_exceptions("DROP TABLE IF EXISTS session_replay_events_v2_test", node_roles=[NodeRole.DATA, NodeRole.COORDINATOR]),
]
```